cli/lsp/analysis.rs

```diff
@@ -16,13 +16,13 @@ use deno_core::error::AnyError;
 use deno_core::serde::Deserialize;
 use deno_core::serde_json::json;
 use deno_core::ModuleSpecifier;
-use lspower::lsp;
-use lspower::lsp::Position;
-use lspower::lsp::Range;
 use once_cell::sync::Lazy;
 use regex::Regex;
 use std::cmp::Ordering;
 use std::collections::HashMap;
+use tower_lsp::lsp_types;
```

Comment:
```suggestion
use tower_lsp::lsp_types as lsp;
```

Again, to reduce code churn.